python/gui/MaaS/cbv/AbstractDatasetView.py

```diff
@@ -0,0 +1,29 @@
+from abc import ABC
+from django.views.generic.base import View
+from dmod.client.request_clients import DatasetExternalClient
+import logging
+logger = logging.getLogger("gui_log")
+from .DMODProxy import DMODMixin, GUI_STATIC_SSL_DIR
+from typing import Dict
+
+
+class AbstractDatasetView(View, DMODMixin, ABC):
```

Comment:
I get the desire to add blanket implementation here for downstream classes, but given that all methods and properties are implemented, is the use of abstract here to indicate to downstream users that they should inherit from this? Still coming up the speed on the codebase so this might be an idiom ive not picked up on yet.

Reply:
Essentially yes. In particular, to reusably abstract a view that has a dataset client property. But it's not especially meaningful on it's own.